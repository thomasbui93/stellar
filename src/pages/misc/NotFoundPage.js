import React, {Component} from 'react'

export class NotFoundPage extends Component {
  render () {
    return <div className='login-page'>
      Not Found Page
    </div>
  }
}
