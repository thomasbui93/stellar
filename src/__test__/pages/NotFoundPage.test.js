import React from 'react';
import {NotFoundPage} from "../../pages/misc/NotFoundPage"
import renderer from 'react-test-renderer';

describe("Not Found Page", function () {
  it('render correctly', () =>{
    const loader = renderer.create(<NotFoundPage/>);
    let tree = loader.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
