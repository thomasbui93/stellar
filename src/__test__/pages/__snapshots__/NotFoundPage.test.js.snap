// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Page render correctly 1`] = `
<div
  className="login-page"
>
  Not Found Page
</div>
`;
